hash/mac_hdflop.xml

```diff
@@ -5,125 +5,3347 @@ license:CC0
+			<feature name="part_id" value="Disk 11/11" />
+			<dataarea name="flop" size="4225984">
+				<rom name="lost in time parts 1 and 2 disk 11.mfm" size="4225984" crc="00626ee8" sha1="87b70f342762eb88a3e625e43914aadae8660217" />
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- Seems to miscalculate the space needed for instalation in anything other than a 68000 CPU -->
+	<software name="mactools30a" supported="partial">
+		<description>Central Point MacTools 3.0a</description>
+		<year>1993</year>
+		<publisher>Central Point Software</publisher>
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk #1" />
+			<dataarea name="flop" size="1474560">
+				<rom name="MacTools Disk 1.img" size="1474560" crc="d1add1a5" sha1="a25d3fd52a1321a9a8ed8216369ea60bb527e4ac" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk #2" />
+			<dataarea name="flop" size="1474560">
+				<rom name="MacTools Disk 2.img" size="1474560" crc="454432b1" sha1="255c33034e64af6b1d4414bdb1b2b4bcb433f6d9" />
+			</dataarea>
+		</part>
+		<part name="flop3" interface="floppy_3_5">
+			<feature name="part_id" value="Disk #3" />
+			<dataarea name="flop" size="1474560">
+				<rom name="MacTools Disk 3.img" size="1474560" crc="ccd3b60c" sha1="8f732b7bcf93870e543391ed8a98f85c21930fd4" />
+			</dataarea>
+		</part>
+		<part name="flop4" interface="floppy_3_5">
+			<feature name="part_id" value="Disk #4" />
+			<dataarea name="flop" size="819200">
+				<rom name="MacTools Disk 4.img" size="819200" crc="19a6266d" sha1="c8b45e9ef549841261628a8aa9c779ded0a10141" />
+			</dataarea>
+		</part>
+		<part name="flop5" interface="floppy_3_5">
+			<feature name="part_id" value="Optimizer Disk" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Optimizer Disk.img" size="1474560" crc="97db31b7" sha1="e3acd588b03b5356babae472949952081dcd7bb1" />
+			</dataarea>
+		</part>
+		<part name="flop6" interface="floppy_3_5">
+			<feature name="part_id" value="Emergency Disk" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Emergency Disk.img" size="1474560" crc="ebf6c551" sha1="3ef9200e4df3ff8dfe7ee9e7e05843f9c1021197" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="macwp15v1">
+		<description>MacWrite Pro 1.5v1</description>
+		<year>1993</year>
+		<publisher>Claris</publisher>
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 1" />
+			<dataarea name="flop" size="1474560">
+				<rom name="disk01.img" size="1474560" crc="c7b3a7a3" sha1="3f822f72bf15e301c4590db6090753a698301b3d" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2" />
+			<dataarea name="flop" size="1474560">
+				<rom name="disk02.img" size="1474560" crc="3faeb185" sha1="bc9bc42e1835cfbf7390e3dd5a95083e888c704e" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="macwp15v3">
```

Comment:
Should the different revisions of 1.5 be clones of the latest one?

Reply:
Makes sense, modified it so macwp15v1 is a clone now.